CoreInteractiveGraph/test/unit/src/au/gov/asd/tac/constellation/graph/interaction/animation/actions/AnimateStopActionNGTest.java

```diff
@@ -54,9 +68,16 @@ public void tearDownMethod() throws Exception {
     @Test
     public void testUpdateValue() {
         System.out.println("AnimateStopAction updateValue");
-        AnimateStopAction instance = new AnimateStopAction();
+        AnimateStopAction instance = spy(new AnimateStopAction());
+        GraphNode gnMock = mock(GraphNode.class);
```

Comment:
not sure if required for a mock, but it might be worth checking whether an object is actually being created here because if so, you probably need to destroy the graph node at the end of the test.

If nothing is created, feel free to ignore this

Reply:
It's needed to get into the if condition, so I added gnMock.destroy()